projects/exui-common-lib/src/lib/services/find-person/find-person.service.ts

```diff
@@ -14,4 +15,27 @@ export class FindAPersonService {
   public find(searchOptions: SearchOptions): Observable<Person[]> {
     return this.http.post<Person[]>('/workallocation2/findPerson', { searchOptions });
   }
+
+  public getSpecificCaseworkers(searchOptions: SearchOptions): Observable<Person[]> {
+    return this.http.get<any[]>('/workallocation2/caseworker').pipe(map(caseworkers => {
+      const people = this.changeToPeople(caseworkers);
+      return people.filter(person => person.name.toLowerCase().includes(searchOptions.searchTerm.toLowerCase()));
```

Comment:
add a check to for person && person.name
```suggestion
      return people.filter(person => person.name.toLowerCase().includes(searchOptions.searchTerm.toLowerCase()));
```